2.0/TextureEncodingTest/README.md

```diff
@@ -0,0 +1,23 @@
+# Texture Encoding Test
+
+## Screenshot
+
+![screenshot](screenshot/screenshot.png)
+
+## Description
+
+This asset tests that textures are sampled with correct transfer functions: base color and emissive must be decoded from sRGB while metallic-roughness values must be passed as-is.
+
+Additionally, this asset tests that gamma values and/or ICC profiles embedded with image files are ignored as the glTF spec requires.
+
+For each of three material slots, four sample models are provided that use different representations of the same effective material properties.
+
+* Models from the first (leftmost) column use only JSON-stored factors to define their materials.
+
+* Models from the second column represent the same materials using 1x1 textures with no color space metadata.
```

Comment:
Do 1x1 textures work everywhere?

Reply:
Yes. It's a power of two: 2<sup>0</sup>=1.

Reply:
I have this memory that we had to increase the minimum texture size to avoid some hardware that didn't support smaller than 4x4 images, but maybe that's no longer an issue.

Reply:
I wonder how that hardware would handle mips as they have to eventually reach 1x1.

Reply:
Not sure. Anyways, if it works, then I'm fine.